scenariolib/event_pageview.go

```diff
@@ -61,11 +61,11 @@ func (view *ViewEvent) send(v *Visit) error {
 	Info.Printf("Sending ViewEvent rank=%d ", view.ClickRank+1)
 
 	event := ua.NewViewEvent()
-	event.Location = v.LastResponse.Results[view.ClickRank].ClickURI
```

Comment:
I was getting error on these lines, but now Travis is having issues with my changes.
Maybe related to different versions of Go. Travis is using 1.5 and I used 1.9

Reply:
Travis is using 1.6 and the tip so latest.

Reply:
Your build is failing because you are not up to date with master.

event.Location is valid, it is the new name of the param in Usage Analytics view request.